lib/common/http3client.c

```diff
@@ -132,6 +132,7 @@ struct st_h2o_http3client_req_t {
 static int handle_input_expect_data_frame(struct st_h2o_http3client_req_t *req, const uint8_t **src, const uint8_t *src_end,
                                           int err, const char **err_desc);
 static void start_request(struct st_h2o_http3client_req_t *req);
+static void destroy_request(struct st_h2o_http3client_req_t *req);
```

Comment:
Declaration here is `static` but the definition at the bottom is non-static?

Reply:
Thank you for the review.

Function definition below inherits the properties of the function declaration here, meaning that it would be static. The code-base tends to not repeat the properties, though I won't mind people doing otherwise.

Reply:
Ah I see. I was not familiar with the fact that it was possible to do this. We can keep this as is.